kolibri/content/utils/channels.py

```diff
@@ -1,26 +1,133 @@
 import fnmatch
 import logging as logger
 import os
+import psutil
 import uuid
+import sqlite3
+from collections import namedtuple
+from django.conf import settings
 
 logging = logger.getLogger(__name__)
 
+KolibriExportedChannelData = namedtuple(
+    "KolibriExportedChannelData",
+    ["name", "id", "path"],
+)
+
+DriveData = namedtuple(
+    "DriveData",
+    [
+        "kind",
+        "name",
+        "id",
+        "writeable",
+        "has_content",
+        "channels",
+    ]
+)
+
+
 def _is_valid_hex_uuid(uuid_to_test):
     try:
         uuid_obj = uuid.UUID(uuid_to_test)
     except ValueError:
         return False
     return uuid_to_test == uuid_obj.hex
 
-def get_channel_id_list_from_scanning_content_database_dir(content_database_dir):
+def get_channel_id_list_from_scanning_content_database_dir(content_database_dir, return_full_dir=False):
```

Comment:
Given it's called `get_channel_id_list...`, feels a bit funny to have it doing the work of constructing these paths. Perhaps that could be done externally, or in a "wrapper" function like this:
```python
def get_content_database_file_paths(content_database_dir):
    full_dir_template = os.path.join(content_database_dir, "{}.sqlite3")
    channel_ids = get_channel_id_list_from_scanning_content_database_dir(content_database_dir)
    return [full_dir_template.format(f) for f in channel_ids]
```